fuzon-http/src/main.rs

```diff
@@ -37,16 +40,13 @@ impl AppState {
         let collections = data
             .collections
             .into_iter()
-            .map(|(k, v)| (k, TermMatcher::from_paths(vec![&v]).unwrap())).collect();
+            .inspect(|(k, _)| println!("Loading {}...", k))
```

Comment:
**suggestion(non-blocking, nitpick)**: since we call them "collections" I would also print "Loading collection {}..."
```suggestion
            .inspect(|(k, _)| println!("Loading collection {}...", k))
```